bin/check-haproxy.rb

```diff
@@ -223,7 +223,7 @@ def acquire_services #rubocop:disable all
     end
 
     parsed = CSV.parse(out, skip_blanks: true)
-    keys = parsed.shift.reject(&:nil?).map { |k| k.match(/(\w+)/)[0].to_sym }
+    keys = parsed.shift.reject(&:nil?).map { |k| k.match(/([\-\w]+)/)[0].to_sym }
```

Comment:
From a backwards compatibility perspective I think we need to consider the `\-` optional if I understand correctly. Does that make sense? I have not spent much time with HAProxy lately so my knowledge may be out of date.
```suggestion
    keys = parsed.shift.reject(&:nil?).map { |k| k.match(/([(\-)?\w]+)/)[0].to_sym }
```

Reply:
Unfortunately i do not remember details, it's almost year old and i wrote golang replacement for it.

Reply:
Suggested change works fine with HAProxy v1.8.x and v2.4.x